pkg_ci.Rmd

```diff
@@ -102,7 +102,7 @@ You also need to give Codecov access to your github repository, see [Codecov qui
 Then add a Codecov status badge to the top of your README.md, see [Codecov status badges](https://docs.codecov.com/docs/status-badges).
 
```

Comment:
For PT translation https://github.com/ropensci/dev_guide/pull/824/files

Reply:
Hi @maelle ! Sorry for taking so long. I have checked, and @fblpalmeira did this review.

Reply:
Hi Maelle! It's on main: https://github.com/ropensci/dev_guide/blob/4260c92c9e23d8eb764e0dc59ae8990f3dc82f51/pkg_ci.pt.Rmd#L91

Reply:
Awesome thank you